src/pages/products/ProductFilter.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+
+const ProductFilter = () => {
+    return (
+        <aside className="sidebar flex flex-column gap-1 ml-1 p-1 box-shadow">
+            <div className="flex justify-between align-center ml-1">
+                <h3 className="side-heading">Filters</h3>
+                <button className="clear">Clear</button>
+            </div>
+            <div className="side-heading font-size-large ml-1 bold-500">
+                <h4 className="pb-1">Price</h4>
+                <div className="slider-value">
+                    <span>100</span>
+                    <span>500</span>
+                    <span>1000</span>
+                </div>
+                <input
+                    type="range"
+                    id="vol"
+                    name="vol"
+                    min="100"
+                    max="1000"
+                    step="100"
+                    value="500"
```

Comment:
you can search about defaultValue={}. You will need when taking the input on change.